.github/cilium-actions-master.yml

```diff
@@ -0,0 +1,37 @@
+project: "https://github.com/cilium/cilium/projects/80"
```

Comment:
At the moment, this project tracks high-level goals for the release, not individual PRs. Am I correct in understanding that this action will now cause all PRs to now be part of that project?

Maybe this is OK if there is a clear distinction between the columns that are manually managed by humans (as set during release planning etc.) vs. automated columns for PRs, controlled by this script. What I'm concerned about is just making sure that a human can assess the status of a release fairly quickly by glancing at the project, not getting overwhelmed by too many issues/PRs.

Reply:
>At the moment, this project tracks high-level goals for the release, not individual PRs. Am I correct in understanding that this action will now cause all PRs to now be part of that project?

Part of the project yes but  part of an automated column called "In progress", that column is not will not be managed by humans since it's automated, it's just to give us an overview what PRs are currently outstanding for v1.7